fvm/state/accounts.go

```diff
@@ -134,10 +134,6 @@ func (a *Accounts) GetPublicKeyCount(address flow.Address) (uint64, error) {
 		return 0, newLedgerGetError(keyPublicKeyCount, address, err)
 	}
 
-	if countBytes == nil {
```

Comment:
We removed this because `!countInt.IsUint64()` will catch it?

Reply:
`countInt := new(big.Int).SetBytes(countBytes)` with `len(countBytes) == 0` will just result in `countInt` being zero, so it'll have the same effect